src/pages/community-partners.astro

```diff
@@ -0,0 +1,97 @@
+---
+import Layout from '@layouts/Default.astro'
+
+const partners = [
+  {
+    'name': 'The Algorithms',
+    'image': 'https://github.com/TheAlgorithms.png',
+    'link': 'https://the-algorithms.com',
+    'description': 'Open Source resource for learning Data Structures & Algorithms and their implementation in any Programming Language'
+  },
+  {
+    'name': 'GitHub',
+    'image': 'https://github.com/GitHub.png',
+    'link': 'https://github.com',
+    'description': 'GitHub is one of the world’s largest community of developers. It’s an intricate platform that fosters collaboration and communication between developers.'
+  },
+  {
+    'name': 'Web3Community',
+    'image': 'https://github.com/Web3Community.png',
+    'link': 'https://web3community.github.io',
+    'description': 'An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀'
+  }
+]
+---
+
+<Layout>
+<div class="mt-20 max-w-full text-center">
+  <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Community Partners</h2>
+  <h4 class="mx-auto w-full font-display text-xl m-4 p-8 dark:text-gray-100">Community Partners help us educate people about web3 and, become a part of their creator success</h4>
```

Comment:
I don't think that comma is needed there, looks wrong, IMO.
```suggestion
  <h4 class="mx-auto w-full font-display text-xl m-4 p-8 dark:text-gray-100">Community Partners help us educate people about web3 and become a part of their creator success</h4>
```

Reply:
I tried adding your suggestion, Grammarly blocked me to do so. There is a comma after and in British English afaik, not sure about American